geometry/grassmann.hpp

```diff
@@ -37,7 +37,8 @@ class Multivector<Scalar, Frame, 1> {
   Scalar Norm() const;
 
   template<typename S>
-  void Orthogonalize(Multivector<S, Frame, 1>& multivector) const;
+  Multivector OrthogonalizedAgainst(
+      Multivector<S, Frame, 1> const& multivector) const;
```

Comment:
well, we need template parameters, this isn't the same type as this `Multivector`. We could go with `Vector`, but I'm not sure it would make things clearer (unless we also change the return type to `Vector<Scalar, Frame>`, but then that's even more template parameters...).